### PR TITLE
fix(web): language menu key highlighting

### PR DESCRIPTION
**Pull request by jahorton**
Fixes point 3 of #2990.

I'm kind of surprised the fix turned out to be so small.  I expected there to be extra changes required to drop the highlighting after the language menu is dismissed... but in my tests, this seems to automatically be handled?  Quite a pleasant surprise, assuming this holds on all affected platforms.

## User Testing

@MakaraSok 

I've tested a fair bit with iOS and through Chrome's mobile device emulation, but could also use some extra tests - especially for the Android app.  (Both in-app and as system keyboard)

Main things to check:
 - Does globe key highlighting 'feel' right?
 - Does it go away after the language-selection menu has been dismissed?

### CODE REVIEW

**Comment by MakaraSok** [comment]
@darcywong00 When using as a system keyboard, it feels a little off when the globe key is clicked on. The keyboard disappears and the language-selection menu is shown.

Screenshot_1611213066

Now, compare with the in-app where the keyboard stays in the background with the globe key highlighted.

Screenshot_1611213405

These behaviors are consistent on both emulator and physical device.

**Comment by jahorton** [comment]
I'm fairly certain that's the standard Android behavior, though I'll have to defer to @darcywong00 to confirm.  Either way... that's something this PR doesn't touch.  So, it sounds like things are fine regarding this PR's changes, at least.

**Comment by darcywong00** [comment]
> when the globe key is clicked on. The keyboard disappears and the language-selection menu is shown.

I haven't paid much attention to the background. But apparently, the Keyman System keyboard has always done this (as of version 2.8.300)

**Comment by MakaraSok** [comment]
Samsung and Gboard keyboards show the keyboard menu on top of the keyboard layout, just like the behavior we have in our in-app.

**Comment by jahorton** [comment]
... sounds like an issue.  Now, whether or not we tackle it in 14.0 beta... that's another question.

**Comment by jahorton** [comment]
Went ahead and made a basic issue for it.  Ended up going with "fix", since from the standard Android experience, I guess it would "feel like a bug".

File: web/source/osk/visualKeyboard.ts [hunk 1]
```diff
@@ -1066,9 +1066,13 @@ namespace com.keyman.osk {
...
 
       // Special function keys need immediate action
+      let _this = this;
       if(keyName == 'K_LOPT' || keyName == 'K_ROPT')      {
         window.setTimeout(function(this: VisualKeyboard){
           PreProcessor.clickKey(key);
+          // Because we immediately process the key, we need to re-highlight it after the click.
+          _this.highlightKey(key, true);
+          // Highlighting'll be cleared automatically later.
         }.bind(this),0);
```
Comment (mcdurdin) [inline web/source/osk/visualKeyboard.ts:1076]: given we bind `this`, do we need `_this`?

**Review by mcdurdin — APPROVED** [review]

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 14.0.228-beta](https://keyman.com/downloads/releases/beta/14.0.228)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.9-alpha](https://keyman.com/downloads/releases/alpha/15.0.9)

**Pull request merged**